app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -82,14 +82,22 @@ function initializePodcastPlayback() {
 
   function spinPodcastRecord(customMessage) {
     if (audioExistAndIsPlaying() && recordExist()) {
-      getById(`record-${window.activeEpisode}`).classList.add('playing');
+      var podcastPlaybackButton = getById(`record-${window.activeEpisode}`);
+      podcastPlaybackButton.classList.add('playing');
+      var isPressed =
+        podcastPlaybackButton.getAttribute('aria-pressed') === 'true';
+      podcastPlaybackButton.setAttribute('aria-pressed', !isPressed);
       changeStatusMessage(customMessage);
     }
   }
 
   function stopRotatingActivePodcastIfExist() {
     if (window.activeEpisode && getById(`record-${window.activeEpisode}`)) {
-      getById(`record-${window.activeEpisode}`).classList.remove('playing');
+      var podcastPlaybackButton = getById(`record-${window.activeEpisode}`);
+      podcastPlaybackButton.classList.remove('playing');
+      var isPressed =
+        podcastPlaybackButton.getAttribute('aria-pressed') === 'true';
+      podcastPlaybackButton.setAttribute('aria-pressed', !isPressed);
```

Comment:
```suggestion
      podcastPlaybackButton.setAttribute('aria-pressed', 'false');
```
(same as above, I don't think there's a need to check the current attribute state)